vllm/v1/worker/gpu_model_runner.py

```diff
@@ -2569,6 +2575,15 @@ def get_kv_cache_spec(self) -> dict[str, KVCacheSpec]:
         mamba_layers = get_layers_from_vllm_config(self.vllm_config,
                                                    MambaMixer2)
         if len(mamba_layers) > 0:
+            if len(attn_layers) > 0:
+                # Mamba state must be padded to an integer number of
+                # 16th tokens worth of attention pages
+                attn_layer_name = next(iter(attn_layers))
+                attn_page_size = kv_cache_spec[attn_layer_name].page_size_bytes
+                multiple_of = 16 * attn_page_size // block_size
```

Comment:
Why do we need 16 when the `block_size` is large enough? Is this logic better?
```
if attn_page_size < mamba_page_size:
    # suggest a block_size that is a multiple of 16
else:
    # use the current block_size to generate kv_cache_spec
```

Reply:
It's better yep, will change it.

Reply:
Done